languagetool-language-modules/ru/src/main/resources/org/languagetool/resource/ru/tagset.txt

```diff
@@ -530,7 +530,7 @@ right column is LanguageTool's POS tag:
 юф    Ord:PL:D
 юх    Ord:PL:V
 Лу    Ord:PL:V
-юц    Ord:PL:T
+6юц    Ord:PL:T
```

Comment:
Is this by accident? If not, I suggest making a separate PR for this.

Reply:
This is defiinitely by accident. I did not change any Russian language file.